rest-client/src/main/java/org/thingsboard/rest/client/RestClient.java

```diff
@@ -3241,9 +3241,9 @@ public BulkImportResult<Edge> processEdgesBulkImport(BulkImportRequest request)
                 }).getBody();
     }
 
-    public Optional<EdgeInstallInstructions> getEdgeDockerInstallInstructions(EdgeId edgeId) {
-        ResponseEntity<EdgeInstallInstructions> edgeInstallInstructionsResult =
-                restTemplate.getForEntity(baseURL + "/api/edge/instructions/{edgeId}", EdgeInstallInstructions.class, edgeId.getId());
+    public Optional<EdgeInstructions> getEdgeDockerInstallInstructions(EdgeId edgeId, String method) {
+        ResponseEntity<EdgeInstructions> edgeInstallInstructionsResult =
+                restTemplate.getForEntity(baseURL + "/api/edge/instructions/install/{edgeId}/{method}", EdgeInstructions.class, edgeId.getId(), method);
         return Optional.ofNullable(edgeInstallInstructionsResult.getBody());
     }
```

Comment:
please add method getEdgeUpgradeInstructions.
And please rename  this to getEdgeInstallInstructions